src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -493,18 +492,21 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
       */
     def calculateStake(votingData: Seq[VotingData]): Future[BigDecimal] =
       Future
-        .sequence(
+        .traverse(
```

Comment:
oops, I thought it might become something like
```scala
Future.traverse(votingData){
  case VotingData(...) =>
    tezosNodeOperator.getAccountBalanceForBlock(BlockHash(hash), AccountId(...))
}.map(...)
```?

Reply:
oh, I get it, changed 😄